### ISSUE TITLE
http://link.tl/12lyE

### ISSUE THREAD

**Issue by ardacihaner** [reek/anti-adblock-killer#3265]
Aaklist and Aakscript are enabled. 

Using Ublock Origin on Chrome

Example url: http://link.tl/12om7

**Comment by uBlock-user** [comment]
add 

```
link.tl##.adblockBox
link.tl##.adblockOverlay
```

@jspenguin2017 There's a 6 sec timer running if you can patch or simply bypass the website as there's no recaptcha.

**Comment by jspenguin2017** [comment]
@uBlock-user reCaptcha has explicit mode and stealth mode. You can't assume there is no reCaptcha just because you don't see it. 
![image](https://cloud.githubusercontent.com/assets/7283682/26755446/9b887810-484a-11e7-90d3-eb2273356afd.png)

**Comment by uBlock-user** [comment]
@jspenguin2017 So did you find any ?

**Comment by uBlock-user** [comment]
Also it's not making any connection to Google's servers, so I doubt you will find it at all.

**Comment by jspenguin2017** [comment]
Yea, looks like it's disabled, it seems to have some code for it. But it still does a POST, so it's a pain to bypass.

**Comment by uBlock-user** [comment]
Well just kill the timer if you can. URL shorteners are always full of shit. Some people still insist on using it which is beyond me.

**Comment by jspenguin2017** [comment]
There are legit reasons to use URL shorteners, https://github.com/jspenguin2017/jspenguin_com/blob/master/r.php
Like when your announcement panel is too small. 

But I haven't found legit third party shorteners though, they have at least some form of tracking.

**Comment by ale5000-git** [comment]
@jspenguin2017: It have a weird behaviour with the fix.
In certain cases it decrement fasterly from 6 to 0 many times.
When I click Skip it go to another Skip, and at the second Skip it show a dialog asking if exit from the site.

**Comment by uBlock-user** [comment]
Those are like one in thousand which makes it a rare case and exceptional case and all of them have tracking. I have yet to find any that doesn't make any kind of third-party connection.

**Comment by uBlock-user** [comment]
> When I click Skip it go to another Skip, and at the second Skip it show a dialog asking if exit from the site.

Yes it's by design. Nothing to do with the fix. A cosmetic filter can't fasten the timer or slow it It only hides the anti adblock banner, nothing else.

**Comment by jspenguin2017** [comment]
~~I think I can remove the popup dialogue.~~

**Comment by jspenguin2017** [comment]
NVM, can't seem to be able to bind onto a good event.
